lib/pierre.py

```diff
@@ -28,8 +28,11 @@ def check(payload, headers, host):
         return reply
 
     owner, repo = get_owner_and_repo(payload)
+    dependency_id = get_dependency_id(payload)
```

Comment:
I would like to have this renamed, since this isn't really the dependency id, it's the pull request id. Something in the likes of `pull_request_id` or `root_id`?